test/php/application/forms/Config/UserBackendReorderFormTest.php

```diff
@@ -17,6 +17,11 @@ public function save()
         self::$newConfig = $this->config;
         return false;
     }
+
+    public function setCustomBackend()
+    {
+        return false;
+    }
```

Comment:
Remove that. Setup a mock for the functionality the test requires instead.

Add the following to the test case where the request object is already being mocked:

`Icinga::app()->shouldReceive('getModuleManager->getLoadedModules')->andReturn([]);`

Reply:
I didn't know that mockery allows to concatenate methods in shouldReceive. Good to know...

Reply:
It's called demeter chains :grin: